packages/svelte-docgen/src/parser/mod.js

```diff
@@ -284,7 +292,11 @@ class Parser {
 			return { kind, subkind: "boolean", value: type.intrinsicName === "true" };
 		}
 		if (type.flags & ts.TypeFlags.UniqueESSymbol) {
-			return { kind, subkind: "symbol" };
+			return {
+				kind,
+				subkind: "symbol",
+				alias: this.#checker.getFullyQualifiedName(type.aliasSymbol || type.symbol),
```

Comment:
Huh, this one puzzles me. How `unique symbol` _(I think that's how it looks in the IDE)_ can have an alias?

Reply:
Honestly, I’m not sure what the best approach is here. A symbol alone doesn’t provide much meaning, so I tried recording an alias name for it, but it might not be a good idea.

Reply:
Removed this for now.